omegaconf/dictconfig.py

```diff
@@ -515,10 +514,11 @@ def __contains__(self, key: object) -> bool:
             try:
                 self._resolve_with_default(key=key, value=node)
                 return True
-            except UnsupportedInterpolationType:
-                # Value that has unsupported interpolation counts as existing.
+            except InterpolationResolutionError:
+                # Interpolations that fail count as existing.
                 return True
-            except (MissingMandatoryValue, KeyError, InterpolationResolutionError):
+            except MissingMandatoryValue:
+                # Missing values count as *not* existing.
```

Comment:
Once this settles down we can evaluate this behavior separately.

I want to see if causing `"foo" in OmegaConf.create({"foo": "???"})` return True will have any significant implications to Hydra.

Reply:
Sounds good, resolving this point.